app/scripts/controllers/HomeCtrl.js

```diff
@@ -1,8 +1,13 @@
 (function() {
     function HomeCtrl(Room) {
         this.rooms = Room.all;
+        this.addRoom = function() {
```

Comment:
There's no reason to wrap `this.submit` in this addition function. `this.submit` is called onclick and you have injected `Room` already, so `this.submit` can just call `Room.add`.